packages/front-end/components/Layout/SidebarLink.tsx

```diff
@@ -40,7 +42,7 @@ export type SidebarLinkProps = {
 
 const SidebarLink: FC<SidebarLinkProps> = (props) => {
   const { permissions, superAdmin } = useUser();
-  const { project } = useDefinitions();
+  const { project, segments } = useDefinitions();
```

Comment:
question: Does this mean that we are now loading all segments on all page loads?

Or maybe we were already doing that but just not surfacing it?

The question here is around performance in case a company has lots of segments.